modules/dexertoBidAdapter.md

```diff
@@ -0,0 +1,39 @@
+# Overview
+
+```
+Module Name: Dexerto Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: niels.claes@dexerto.com
+```
+
+# Description
+
+Dexerto currently supports the BANNER type ads through prebid js
+
+Module that connects to dexerto's demand sources.
+
+# Banner Test Request
+```
+  var adUnits = [
+    {
+      code: 'display-ad',
+      mediaTypes: {
+        banner: {
+            sizes: [[300, 250]], 
+        }
+      }
+      bids: [
+        {
+            bidder: 'dexerto',
+            params: {
+                placement_id: 110003,  // Required parameter
+                width: 300,   // Optional parameter 
+                height: 250, // Optional parameter
+                domain: '', // Optional parameter
```

Comment:
I don't see `params.domain` referenced in your adapter code, I also don't think we'd want a publisher to hard code a domain in a request to your adapter. Can this be removed?

Reply:
@pranavsheth please remove this here and your website PR